x-pack/plugins/canvas/public/state/actions/elements.js

```diff
@@ -186,7 +186,12 @@ export const fetchAllRenderables = createThunk(
       fetchElementsOnPages([currentPage]).then(() => dispatch(args.inFlightComplete()));
     } else {
       fetchElementsOnPages([currentPage])
-        .then(() => fetchElementsOnPages(otherPages))
+        // .then(() => fetchElementsOnPages(otherPages))
```

Comment:
++ I like that we're loading the pages sequentially after the current page is loading now. Can you remove this comment? I don't think it's necessary anymore

Reply:
Sure thing.